### PR TITLE
Ajdust open_tablet_view method to 16.0

### PR DESCRIPTION
**Pull request by laszlokr**

### CODE REVIEW

File: mrp_workorder_disable_autostart/models/mrp_workorder.py [hunk 1]
```diff
@@ -9,35 +9,24 @@ class MrpProductionWorkcenterLine(models.Model):
...
-                    "from_manufacturing_order": self.env.context.get(
-                        "from_manufacturing_order"
-                    ),
-                },
+            action = self.env["ir.actions.actions"]._for_xml_id(
+                "mrp_workorder.tablet_client_action"
+            )
+            action["target"] = "fullscreen"
+            action["res_id"] = self.id
+            action["context"] = {
```
Comment (janikvonrotz) [inline mrp_workorder_disable_autostart/models/mrp_workorder.py:25]: This overwrites the entire context.

Try `action["context"]["from_production_order"] = `

**Review by janikvonrotz — CHANGES_REQUESTED** [review]

File: mrp_workorder_disable_autostart/models/mrp_workorder.py [hunk 2]
```diff
@@ -9,35 +9,29 @@ class MrpProductionWorkcenterLine(models.Model):
...
-                    ),
-                },
-            }
+            action = self.env["ir.actions.actions"]._for_xml_id(
+                "mrp_workorder.tablet_client_action"
+            )
+            action["target"] = "fullscreen"
+            action["res_id"] = self.id
+
+            if not isinstance(action.get("context"), dict):
```
Comment (janikvonrotz) [inline mrp_workorder_disable_autostart/models/mrp_workorder.py]: Context is always a dict. What is the goal of this check?

Comment (laszlokr) [reply mrp_workorder_disable_autostart/models/mrp_workorder.py]: Without this check there is an error:
```
models/mrp_workorder.py", line 26, in open_tablet_view
    action["context"]["active_id"] = self.id
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
TypeError: 'str' object does not support item assignment
```

Comment (janikvonrotz) [reply mrp_workorder_disable_autostart/models/mrp_workorder.py]:
```python
# Create and return action as defined in open_tablet_view
     action = self.env["ir.actions.actions"]._for_xml_id("mrp_workorder.tablet_client_action")
        action['target'] = 'fullscreen'
        action['res_id'] = self.id
        action['context'] = {
            'active_id': self.id,
            'from_production_order': self.env.context.get('from_production_order'),
            'from_manufacturing_order': self.env.context.get('from_manufacturing_order')
        }
        return action
```

**Review by janikvonrotz — APPROVED** [review]

**Pull request merged**